Description: update upstream copyright date
 Update the upstream copyritght date wrt NEWS.
Origin: debian
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2014-08-06

--- a/sbin/firehol.in
+++ b/sbin/firehol.in
@@ -4,8 +4,8 @@
 #
 #   Copyright
 #
-#       Copyright (C) 2003-2013 Costa Tsaousis <costa@tsaousis.gr>
-#       Copyright (C) 2012-2013 Phil Whineray <phil@sanewall.org>
+#       Copyright (C) 2003-2014 Costa Tsaousis <costa@tsaousis.gr>
+#       Copyright (C) 2012-2014 Phil Whineray <phil@sanewall.org>
 #
 #   License
 #
@@ -53,8 +53,8 @@
 	${CAT_CMD} <<EOF
 
 FireHOL $VERSION
-(C) Copyright 2003-2013 Costa Tsaousis <costa@tsaousis.gr>
-(C) Copyright 2012-2013 Phil Whineray <phil@firehol.org>
+(C) Copyright 2003-2014 Costa Tsaousis <costa@tsaousis.gr>
+(C) Copyright 2012-2014 Phil Whineray <phil@firehol.org>
 FireHOL is distributed under the GPL v2+.
 Home Page: http://firehol.org
 
--- a/sbin/fireqos.in
+++ b/sbin/fireqos.in
@@ -4,7 +4,7 @@
 #
 #   Copyright
 #
-#       Copyright (C) 2003-2013 Costa Tsaousis <costa@tsaousis.gr>
+#       Copyright (C) 2013-2014 Costa Tsaousis <costa@tsaousis.gr>
 #
 #   License
 #
@@ -3164,7 +3164,7 @@
 
 $cat_cmd <<EOF
 FireQOS $VERSION
-(C) 2013 Costa Tsaousis, GPL
+(C) 2013-2014 Costa Tsaousis, GPL
 
 EOF
 
